engineering_tools/carma

```diff
@@ -339,6 +339,174 @@ carma__stop() {
     fi
 }
 
+carma__swarm() {
+
+    if [ "$1" = "deploy" ]; then
+
+        echo "Checking Docker Swarm status..."
+        if [ "$(docker info | grep Swarm | sed 's/Swarm: //g')" == " inactive" ]; then
+            echo "Initalizing Docker Swarm..."
+            docker swarm init --advertise-addr 192.168.88.100 # consider making a variable
```

Comment:
I think it could be worth having commands for setting the the following:

- Manager's IP address 
- Worker's IP address
- Worker's user name

Reply:
Agreed, already had considered it. Will add.

Important note- the dual-compute PCs have the exact same hostname of `dev@[some model-specific Spectra name]`. This is bad and should be changed but wanted to check for a go-ahead. This may be a pain but I have not looked into it yet.

Reply:
Yeah, commands or an environment variable would be most preferable here. That way we can more easily deploy a swarm setup in places that don't match our vehicle networks (AWS).